resources/windows/windows.pkr.hcl

```
@@ -163,10 +168,9 @@ build {
   provisioner "windows-restart" {
   }
 
-  provisioner "powershell" {
-    elevated_user     = var.install_user
-    elevated_password = var.install_pass
-    scripts            = ["scripts/windows/configs/ssh-auth.ps1"]
+    provisioner "file" {
+    source      = var.authorized_keys_path
+    destination = "C:\\Users\\vagrant\\.ssh\\authorized_keys"
```

```suggestion
    destination = "C:/Users/vagrant/.ssh/authorized_keys"
```

Based on the above snippet in the same file:

```
  provisioner "file" {
    destination = "C:/vagrant"
    source      = "resources"
  }
```